mava/components/jax/training/model_updating.py

```diff
@@ -113,7 +113,7 @@ def model_update_minibatch(
             )
 
             # Update the networks and optimizors.
-            metrics = {}
+            # metrics = {}
```

Comment:
Maybe remove? Or uncomment?

Reply:
Okay! also I will add the type so we don't have mypy error

Reply:
Done!